flash_notebooks/finetuning/image_classification.ipynb

```diff
@@ -0,0 +1,241 @@
+{
```

Comment:
<p>Add: to classify images of ants and bees</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyTorchLightning/lightning-flash/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Done !</p>